blocklylib-core/src/main/java/com/google/blockly/model/Input.java

```diff
@@ -100,7 +100,7 @@
         INPUT_TYPES.add(TYPE_DUMMY_STRING);
     }
 
-    private final String mName;
+    private final @Nullable String mName;
```

Comment:
Dummy inputs, most importantly. 

Per offline conversation, I have updated the code to enforce names. I will also patch web to do the same.